src/vs/workbench/contrib/positronVariables/browser/components/variableItem.css

```diff
@@ -115,4 +115,15 @@
 
 .variable-item .details-column .right-column .viewer-icon {
 	display: flex;
+	cursor: pointer;
+	transition: background-color 0.2s ease, border-color 0.2s ease;
+	border-radius: 4px;
+	padding: 4px;
+	border: 1px solid transparent;
+	background-color: transparent;
+}
+
+.variable-item .details-column .right-column .viewer-icon:hover {
+	border: 1px solid var(--vscode-button-foreground, #ffffff);
+	background-color: var(--vscode-button-background, #0e639c);
```

Comment:
Likely a small nit, but I think we use a color closer to #dae1ea for hover on clickable icons, ie: <code style="color :#dae1ea">#dae1ea</code> according to my local color inspector tool.

<img width="537" alt="image" src="https://github.com/user-attachments/assets/692ab0cc-3a37-4061-a1b0-c1e6a7d44e37" />

Reply:
Got it, thanks!